deployment/user.go

```diff
@@ -17,6 +19,63 @@ var (
 	}
 )
 
+// UserList returns a list of user with deployment access
+func UserList(deploymentId string, email string, userId string, fullName string, client *houston.Client, out io.Writer) error {
+	user := map[string]interface{}{
+		"userId":   userId,
+		"email":    email,
+		"fullName": fullName,
+	}
+	variables := map[string]interface{}{
+		"user":         user,
+		"deploymentId": deploymentId,
+	}
+	req := houston.Request{
+		Query:     houston.DeploymentUserListRequest,
+		Variables: variables,
+	}
+
+	r, err := req.DoWithClient(client)
+	if err != nil {
+		fmt.Println(err)
+		return err
+	}
+	deploymentUsers := r.Data.DeploymentUserList
+
+	if len(deploymentUsers) < 1 {
+		out.Write([]byte(messages.HoustonInvalidDeploymentUsers))
+		return nil
+	}
+
+	header = []string{"USER ID", "NAME", "EMAIL", "ROLE"}
+	tab := printutil.Table{
+		Padding:        []int{44, 50},
+		DynamicPadding: true,
+		Header:         header,
+	}
+
+	// Build rows
+	for _, d := range deploymentUsers {
+		role := filterByRoleType(d.RoleBindings, houston.DeploymentRole)
+		tab.AddRow([]string{d.Id, d.FullName, d.Username, role}, false)
+	}
+
+	tab.Print(out)
+
+	return nil
+}
+
+// filterByRoleType selects the type of role from a list of roles
+func filterByRoleType(roleBindings []houston.RoleBinding, roleType string) string {
+	for _, roleBinding := range roleBindings {
+		if strings.Contains(roleBinding.Role, roleType) {
+			return roleBinding.Role
+		}
+	}
+
+	return ""
+}
```

Comment:
@andriisoldatenko should this utility function go here for roles or should we add this someplace else?  It's meant to filter out roles based on the type `DEPLOYMENT`, `WORKSPACE`, `SYSTEM`, etc.

I was also wondering if it's best practice to return an empty string or throw an error when there is a mismatch with the type?